src/Context/Context.php

```diff
@@ -119,7 +120,13 @@ public function prepareTranslator($path = '/translations/', $withUpdater = false
       }
 
       //Setup the translation logger to listen @ shutdown
-      $cubex->retrieve(TranslationUpdater::class, [$this->getCubex(), $tplCatalog, $catFile, static::DEFAULT_LANGUAGE]);
+      $cubex->retrieve(TranslationUpdater::class, [
+        $this->getCubex(),
```

Comment:
```suggestion
        $cubex,
```

does this allow for a single line again?